File-Formats/DocIO/Convert-Word-to-PDF-in-Windows-Forms.md

```diff
@@ -0,0 +1,113 @@
+
+Step 3: Include the following namespaces in the **Form1.Designer.cs** file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using Syncfusion.OfficeChart
+using Syncfusion.DocToPDFConverter
+using Syncfusion.OfficeChartToImageConverter
+using Syncfusion.Pdf
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 4: Add a new button in **Form1.Designer.cs** to create Word file as follows.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+private Button WordToPDF;
+private Label label;
+
+private void InitializeComponent()
+{
+    label = new Label();
+    WordToPDF = new Button();
+
+    //Label
+    label.Location = new System.Drawing.Point(0, 40);
+    label.Size = new System.Drawing.Size(426, 35);
+    label.Text = "Click the button to view a Word document generated by Essential DocIO. Please note that Microsoft Word Viewer or Microsoft Word is required to view the resultant Word document";
+    label.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
+
+    //Button
+    WordToPDF.Location = new System.Drawing.Point(180, 110);
+    WordToPDF.Size = new System.Drawing.Size(85, 36);
+    WordToPDF.Text = "Convert Word document to PDF";
+    WordToPDF.Click += new EventHandler(WordToPDF_Click);
+
+    //Create Word
+    ClientSize = new System.Drawing.Size(450, 150);
+    Controls.Add(label);
+    Controls.Add(btnWordToPDF);
+    Text = "Convert Word document to PDF";
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 5: Add the following code in **WordToPDF_Click** to **open an existing Word document in Windows-Forms**.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+//Open an existing Word document.
+WordDocument wordDocument = new WordDocument("Input.docx");
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 6: Add below code example to convert the Word document to PDF.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+//Initializes the ChartToImageConverter for converting charts during Word to PDF conversion.
+wordDocument.ChartToImageConverter = new ChartToImageConverter();
+//Creates an instance of the DocToPDFConverter.
+DocToPDFConverter converter = new DocToPDFConverter();
+//Converts the Word document to a PDF document.
+PdfDocument pdfDocument = converter.ConvertToPDF(wordDocument);
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 7: Add below code example to **save the Word document in Windows Forms**.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+//Saves the resulting PDF document to the specified output path.
+pdfDocument.Save("WordToPDF.pdf");
+{% endhighlight %}
+
+{% endtabs %}
+
+By executing the program, you will get the **Word document** as follows.
+
+![Windows Forms output Word document](Windows-Forms_images/ConvertWordtoPDF.png)
```

Comment:
Prepare 1/2 screenshot Word document vs PDF using the input template from https://github.com/SyncfusionExamples/DocIO-Examples/blob/main/Word-to-PDF-Conversion/Convert-Word-document-to-PDF/.NET-Framework/Convert-Word-document-to-PDF/Template.docx

Reply:
Addressed